modules/apps/content-dashboard/content-dashboard-api/src/main/java/com/liferay/content/dashboard/item/action/ContentDashboardItemAction.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.content.dashboard.item.action;
+
+/**
+ * @author David Arques
+ */
+public interface ContentDashboardItemAction {
+
+	public String getLabel();
+
+	public String getName();
+
+	public Type getType();
+
+	public String getURL();
+
+	public enum Type {
+
+		DELETE, EDIT, VIEW, VIEW_IN_PANEL
```

Comment:
@cgoncas  I've added some of the types discussed here https://docs.google.com/document/d/1d9MwzPk0hAG07J67oawu74lIz3ai_upuK-R39yll_Pc/edit#heading=h.8khmimhb97h

What do you think?

Reply:
It is OK, I am not a big fan of add things before using them. But if this is already done, let's keep it.